packages/frontend/src/components/map/Map.tsx

```diff
@@ -0,0 +1,39 @@
+'use client';
+import React, { useEffect } from 'react';
+import { MapContainer, Marker, TileLayer, Popup } from 'react-leaflet';
+// import L from 'leaflet';
+import 'leaflet/dist/leaflet.css';
+
+// const position: [number, number] = [51.505, -0.09];
+
+function Map() {
+  useEffect(() => {
+    if (typeof window !== 'undefined') {
+      require('leaflet');
+    }
+  }, []);
+  return (
+    <MapContainer
+      data-testid="map"
+      style={{
```

Comment:
**nitpick (non-blocking):** Please review the styling convention as we are currently using `TailwindCSS` classNames. This is non-blocking but would be good to stick to stick to a unified system